### PR TITLE
Create 2023-10-26-hacking-postgres-ep6

### PR DESCRIPTION
**Pull request by ericankenman**

### CODE REVIEW

**Comment by ericankenman** [comment]
This still needs the YT embed link; I'll add that tomorrow.

File: blog/2023-10-26-hacking-postgres-ep6 [hunk 1]
```diff
@@ -0,0 +1,709 @@
...
+authors: [eric]
+tags: [postgres, hacking_postgres]
+---
+
+
+In this episode, Ry, Regina, and Paul talk about geospatial development, the challenges of creating and maintaining an extension across multiple Postgres development cycles, and what they’re hoping for in the future of Postgres. 
+
+Watch below, or listen on Apple/Spotify (or your podcast platform of choice). Special thanks to Regina and Paul for joining us today!
+
+
```
Comment (ericankenman) [inline blog/2023-10-26-hacking-postgres-ep6:13]: <iframe width="560" height="315" src="https://www.youtube.com/embed/q3TXM6Nu7Aw?si=El0OVHhXKXkXAoQ1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

Comment (ericankenman) [inline blog/2023-10-26-hacking-postgres-ep6:13]: <iframe width="560" height="315" src="https://www.youtube.com/embed/q3TXM6Nu7Aw?si=El0OVHhXKXkXAoQ1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

Comment (ericankenman) [inline blog/2023-10-26-hacking-postgres-ep6:13]: <iframe width="560" height="315" src="https://www.youtube.com/embed/q3TXM6Nu7Aw?si=El0OVHhXKXkXAoQ1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

**Review by williamong68 — APPROVED** [review]

File: blog/2023-10-26-hacking-postgres-ep6 (hunk 1 above)
Comment (ericankenman) [inline blog/2023-10-26-hacking-postgres-ep6:13]: <iframe width="560" height="315" src="https://www.youtube.com/embed/q3TXM6Nu7Aw?si=El0OVHhXKXkXAoQ1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

**Review by ChuckHend — APPROVED** [review]

**Pull request merged**